pkg/drivers/krunkit/krunkit.go

```diff
@@ -61,6 +61,7 @@ const (
 	serialFileName = "serial.log"
 	logLevelInfo   = "3"
 	defaultSSHUser = "docker"
+	maxAttempts    = 60
```

Comment:
This value makes sense only in Start(), and even there is should be right above the code using it. Putting it here confuses the reader that this driver uses maxAttempts for many operations while it is used only for one operation.

Let make it a variable at the start right above the loop. This way we can bake into the value the multiplier in the 2 version using a multiplier.